### PR TITLE
Use templated import warning instead of contrib.messages

### PR DESCRIPTION
**Pull request by evansd**
The previous approach had issues with showing duplicate warnings
following redirects and this approach, while cruder, is simpler and more
reliable.

### CODE REVIEW

File: openprescribing/openprescribing/settings/base.py [hunk 1]
```diff
@@ -151,7 +151,8 @@
                 'frontend.context_processors.support_email',
                 'frontend.context_processors.google_tracking_id',
                 'frontend.context_processors.api_host',
-                'frontend.context_processors.debug'
+                'frontend.context_processors.debug',
+                'pipeline.context_processors.import_in_progress'
```
Comment (inglesp) [inline openprescribing/openprescribing/settings/base.py]: Add a trailing comma?

Comment (evansd) [reply openprescribing/openprescribing/settings/base.py]: Haha! You know I actually did out of habit and then realised that the list didn't have one previously and so removed it for consistency...

Comment (evansd) [reply openprescribing/openprescribing/settings/base.py]: I'm back on another branch now so I think that might be an enhancement for another day

Comment (inglesp) [reply openprescribing/openprescribing/settings/base.py]: 👍

**Pull request merged**